InfinityCosmos/ForMathlib/AlgebraicTopology/SimplicialSet/Homotopy.lean

```diff
@@ -88,54 +89,81 @@ section
 
 open SimplexCategory
 
-variable {A : SSet.{u}} (f g : Δ[1] ⟶ A)
+variable {A : SSet.{u}} (f g : A _[1])
 
 structure HomotopyL where
-  homotopy : Δ[2] ⟶ A
-  face0 : standardSimplex.map (δ 0) ≫ homotopy =
-    standardSimplex.map (σ 0) ≫ standardSimplex.map (δ 0) ≫ f
-  face1 : standardSimplex.map (δ 1) ≫ homotopy = g
-  face2 : standardSimplex.map (δ 2) ≫ homotopy = f
+  x : A _[2]
+  face0 : A.δ 0 x = A.σ 0 (A.δ 0 f)
+  face1 : A.δ 1 x = g
+  face2 : A.δ 2 x = f
```

Comment:
I have been working with the naming of fields as below. I'm not sure which is preferable (or more in line with the mathlib style).

```lean
structure HomotopyL where
  simplex : A _[2]
  δ₀ : A.δ 0 simplex = A.σ 0 (A.δ 0 f)
  δ₁ : A.δ 1 simplex = g
  δ₂ : A.δ 2 simplex = f
```

Reply:
Naming in Lean has always been mysterious to me. I would prefer a shorter field name if it's called often. `simplex` is too long to me and there seems to be no good abbreviation for it, so I just chose `x`. `δ₀` would definitely better than `face0`.

Reply:
The code looks good, but I think mathlib style conventions lean towards slightly longer names that give a better sense of the data involved. I suggest something like:

```
structure HomotopyL where
  simplex : A _[2]
  δ₀_eq : A.δ 0 simplex = A.σ 0 (A.δ 0 f)
  δ₁_eq : A.δ 1 simplex = g
  δ₂_eq : A.δ 2 simplex = f
```